pkg/model/form.go

```diff
@@ -0,0 +1,56 @@
+package model
+
+import (
+	"fmt"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
```

Comment:
Yes.  Pillar needs a comprehensive commenting pass. I'd like to identify a standard that we can publish to documentation.  

Let's address this at an application level: https://github.com/coralproject/pillar/issues/109

There's also a strong tie in here with Cay and other apps.  The concept of Form needs to live somewhere common in the Coral Ecosystem.